src/sentry/integrations/github/integration.py

```diff
@@ -100,14 +100,17 @@ def search_issues(self, query):
 
     def get_stacktrace_link(self, repo, filepath, default_version):
         try:
-            resp = self.get_client().get_file_url(repo.name, filepath)
+            self.get_client().check_file(repo.name, filepath)
         except ApiError as e:
             if e.code != 404:
                 raise
             return None
 
-        # if it exists return the url
-        return resp["html_url"]
+        # Must format the url ourselves since `check_file` is a head request
+        # "https://github.com/octokit/octokit.rb/blob/master/README.md"
+        web_url = "https://github.com/{}/blob/{}/{}".format(repo, default_version, filepath)
```

Comment:
@MeredithAnya if you're going to use `.format`, this should be a unicode string (`u"...`)